tx-pool/src/process.rs

```diff
@@ -650,10 +652,50 @@ impl TxPoolService {
 
     pub(crate) async fn clear_pool(&mut self, new_snapshot: Arc<Snapshot>) {
         let mut tx_pool = self.tx_pool.write().await;
-        let config = tx_pool.config;
+        let config = Arc::clone(&tx_pool.config);
         self.last_txs_updated_at = Arc::new(AtomicU64::new(0));
         *tx_pool = TxPool::new(config, new_snapshot, Arc::clone(&self.last_txs_updated_at));
     }
+
+    pub(crate) async fn cache_pool(&self) -> Result<(), AnyError> {
```

Comment:
This is not a caching behavior, suggest to rename the fn name to `dump_pool` and change the default dump file name to `tx_pool.dump`

Reply:
I will push a commit to fix this.

I remove part of codes when merge develop, so the name is not proper now.